2022-05-09 19:56:33,530 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: http://localhost:4030/webhook/topic/present_proof/; Error: (<class 'aiohttp.client_exceptions.ClientOSError'>, ClientOSError(None, 'Can not write request body for http://localhost:4030/webhook/topic/present_proof/'), <traceback object at 0x7f4398777a48>); Re-queue failed message ...
